client/app/sources/EventSource.js

```diff
@@ -5,6 +5,18 @@ const EVENT_PATH = '/events';
 const EventSource = {
   fetch(params) {
     return axios.get(EVENT_PATH, { params });
+  },
+  remove(id) {
+    const token = document.querySelector('meta[name="csrf-token"]').content;
```

Comment:
Don't do this in the remove method because it will be needed for other requests so just extract it into a function.